packages/graphql-language-service-interface/src/__tests__/getOutline-test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ *  @flow
+ */
+
+import {expect} from 'chai';
+import {describe, it} from 'mocha';
+import {getOutline} from '../getOutline';
+
+describe('getOutline()', () => {
+  it('returns outline trees correctly', () => {
+    const query = `query test {
+      name
+    }`;
+    const tree = getOutline(query).outlineTrees;
+    expect(tree).to.not.be.undefined;
+    expect(tree.length).to.equal(1);
+    expect(tree[0].startPosition.line).to.equal(0);
```

Comment:
I tried this way before but the deep.equal didn't work so well since there's one more function defined in each Position object, besides the line and character attributes. When constructing a new Position object it also constructs a new function a think, so the 'expect' will fail.